text/src/autogluon/text/automm/utils.py

```diff
@@ -125,6 +124,62 @@ def infer_metrics(
     return validation_metric_name, eval_metric_name
 
 
+def get_minmax_mode(metric_name: str):
+    """
+    Get minmax mode based on metric name
+
+    Parameters
+    ----------
+    metric_name
+        A string representing metric
+
+    Returns
+    -------
+    mode
+        The min/max mode used in selecting model checkpoints.
+        - min
+             Its means that smaller metric is better.
+        - max
+            It means that larger metric is better.
+    """
+    assert metric_name in METRIC_MODE_MAP, f"{metric_name} is not a supported metric. Options are: {VALID_METRICS}"
+    return METRIC_MODE_MAP.get(metric_name)
```

Comment:
use metric's [`higher_is_better`](https://torchmetrics.readthedocs.io/en/stable/pages/overview.html#metrics-and-hyperparameter-optimization) instead.